packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -55,6 +56,12 @@ export default class MobileMapListing extends Component {
     this.loadedCarousel = false
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isActive !== this.props.isActive) {
```

Comment:
So this checks only 2 things to forceCheck:
1. active -> inactive
2. inactive -> active

Does that really need to happen? Are you sure this is really doing anything?

Reply:
When I check when it moves in either state it makes sure that the Photo is loaded when the Carousel slides the next three listings. If I don't do a forceCheck here, the lazyloaded placeholder always stays in place regardless of the onSlide call in the parent ListingCarousel's onSlide event.